### PR TITLE
[AUC-3] Typography

### PR DESCRIPTION
**Pull request by Trevoule**
added responsive fonts for the typography default theme

### CODE REVIEW

**Review by sergeevds — CHANGES_REQUESTED** [review]

File: react/src/components/Material/theme.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    Desktop: {
+      fontFamily: "Gilroy",
+      h1: {
+        fontSize: 32,
+        lineHeight: 1.125,
+        "@media (min-width:800px)": {
+          fontSize: 64,
+          lineHeight: 2.25,
+        },
+        // [theme.breakpoints.up("md")]: {
```
[the remark this answers is not in the document]
Comment (Trevoule) [reply react/src/components/Material/theme.js]: fixed

File: react/src/components/Material/theme.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 900,
+      lg: 1200,
+      xl: 1536,
+    },
+  },
+  // components: {
```
[the remark this answers is not in the document]
Comment (Trevoule) [reply react/src/components/Material/theme.js]: fixed

File: react/src/components/Typography/stories.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+import React, { Fragment } from "react";
+
+import { makeStyles } from "@material-ui/styles";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles((theme) => ({
```
[the remark this answers is not in the document]
Comment (Trevoule) [reply react/src/components/Typography/stories.js]: fixed

File: react/src/components/Typography/stories.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      <div>
+        <Typography
+          gutterBottom
+          variant="h1"
+          component="h1"
+          className={classes.DesktopXL}
+        >
+          Desktop Text - Gilroy
+        </Typography>
+        <Typography variant="h2" component="h2" className={classes.DesktopL}>
```
[the remark this answers is not in the document]
Comment (Trevoule) [reply react/src/components/Typography/stories.js]: fixed

File: react/src/components/Material/theme.js [hunk 5]
```diff
@@ -5,16 +5,82 @@ import { typographyTheme } from "../Material/typographyTheme";
 export const theme = createTheme({
   ...paletteTheme,
   ...typographyTheme,
+  breakpoints: {
```
Comment (sergeevds) [inline react/src/components/Material/theme.js:8]: looks like default. If we didn't override it i would get rid of defining

**Review by sergeevds — APPROVED** [review]

File: react/src/components/Material/theme.js (hunk 5 above)
Comment (Trevoule) [reply react/src/components/Material/theme.js:8]: deleted

**Pull request merged**